installer/templates/phx_new/lib/app_name/web.ex

```diff
@@ -0,0 +1,76 @@
+defmodule <%= application_module %>.Web do
+  @moduledoc """
+  A module that keeps using definitions for controllers,
+  views and so on.
```

Comment:
@chrismccord I vaguely remember from your talk (I might be totally off tho) that there were plans to have generators like: `mix phoenix.gen blog comment body:string`, and in `lib/blog.ex` we'd have functions like  `Blog.create_comment` etc.

Have you considered having something like `use MyApp.Web, :controller` but for the Blog module? e.g.:

```elixir
defmodule MyApp.Blog do
  use MyApp.Web, :context
end
```

I guess the `context` macro (or however it could be called) could make the imports/aliases to Repo, Query etc. The obvious problem with the above would be that the `Blog` module should be decoupled from the Web, so the line `use MyApp.Web, ...` would be confusing. Or perhaps the generator would automatically put the aliases/imports into the generated "context" module (but not actually explicitly using the name "context" anywhere in the codebase)? I'm curious if anything like the above would be automatically generated (and thus serve as a convention). P.S. for the time being, in HexWeb we're doing [`use HexWeb.Web, :crud`](https://github.com/hexpm/hex_web/blob/master/web/models/releases.ex#L2)